app/src/styles/StyleCardView.tsx

```diff
@@ -0,0 +1,82 @@
+import { styled } from '@mui/material/styles';
+import { Box, Typography } from '@mui/material';
+import ButtonLink from '@/components/ButtonLink';
+
+// Color Scheme:
+// #ECF2F8: Light blue
+// #96b6d4: Mild blue
+// #477096: Dark blue
+// #4F4846: Black
+// #FDF8F5: White
```

Comment:
Non-blocking: we should move these to constants so that we can easily update the color scheme across our website :) otherwise, we are relying on our team to copy+paste accurately